GDJS/Runtime/runtimescene.js

```diff
@@ -756,3 +762,13 @@ gdjs.RuntimeScene.prototype.getAdhocListOfAllInstances = function() {
     this._constructListOfAllInstances();
     return this._allInstancesList;
 }
+
+/**
+ * Check if the scene was just resumed.
+ * This is true during the first frame after the scene has been unpaused.
+ *
+ * @returns {boolean} The scene was just resumed
+ */
+gdjs.RuntimeScene.prototype.sceneJustResumed = function() {
+    return this._isJustResumed;
+}
```

Comment:
nitpicking: can you configure your editor to always add a new line at the end of files? (it's a weird convention, but better follow it).